lib/presentation/localizations/app_localizations.dart

```diff
@@ -334,6 +334,76 @@ class AppLocalizations {
       args: [numberOfItems],
     );
   }
+
+  String get uploading_files_status_title {
+    return Intl.message(
+      'Uploading files',
+      name: 'uploading_files_status_title',
+    );
+  }
+
+  String get uploading_files_view_button {
+    return Intl.message(
+      'View',
+      name: 'uploading_files_view_button',
+    );
+  }
+
+  String get current_uploads_screen_title {
+    return Intl.message(
+      'Current uploads',
+      name: 'current_uploads_screen_title',
+    );
+  }
+
+  String get current_uploads_my_space_tab {
+    return Intl.message(
+      'My Space',
+      name: 'current_uploads_my_space_tab',
+    );
+  }
+
+  String get current_uploads_shared_space_tab {
+    return Intl.message(
+      'Shared Space',
```

Comment:
Why not use existing strings ?

Reply:
From my opinion, I always want seperate resource responsible and map it to extactly element which use it. It helps me dont have to worry much about relate things when have any update ^^. But don't mind. I will reuse privous resource

Reply:
Ops, unluck :( the previous resource is 'Shared space', we need 'Shared Space'. It may different in other languages, so I think it should be seperated -..-